src/components/select/select-textbox/select-textbox.component.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import Textbox from '../../../__experimental__/components/textbox';
+import OptionsHelper from '../../../utils/helpers/options-helper/options-helper';
+
+const SelectTextbox = ({
+  value,
+  disabled,
+  readOnly,
+  placeholder,
+  onClick,
+  onFocus,
+  onBlur,
+  selectedValue,
+  ...restProps
+}) => {
+  const defaultPlaceholder = I18n.t('select.placeholder', {
+    defaultValue: 'Please Select...'
+  });
+
+  function handleTextboxClick(event) {
+    if (disabled || readOnly) {
+      return;
+    }
+
+    onClick(event);
+  }
+
+  function handleTextboxFocus(event) {
+    if (disabled || readOnly) {
+      return;
+    }
+
+    if (onFocus) {
+      onFocus(event);
+    }
+  }
+
+  function handleTextboxBlur(event) {
+    if (onBlur) {
+      onBlur(event);
+    }
+  }
+
+  function getTextboxProps() {
+    return {
+      placeholder: placeholder || defaultPlaceholder,
+      disabled,
+      readOnly,
+      onClick: handleTextboxClick,
+      onFocus: handleTextboxFocus,
+      onBlur: handleTextboxBlur,
+      ...restProps
+    };
+  }
+
+  return (
+    <Textbox
+      data-element='select-input'
+      inputIcon='dropdown'
+      autoComplete='off'
+      value={ selectedValue }
+      { ...getTextboxProps() }
+    />
+  );
+};
+
+const formInputPropTypes = {
+  /** [Form Field] Id attribute of the input element */
+  id: PropTypes.string,
+  /** [Form Field] Name attribute of the input element */
+  name: PropTypes.string,
+  /** [Form Field] If true the Component will be read-only */
+  readOnly: PropTypes.bool,
+  /** [Form Field] If true the Component will be disabled? */
```

Comment:
I don't think there's supposed to be a question mark here :D

Reply:
nice catch ;)